app/models/comment.rb

```diff
@@ -1,4 +1,10 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_create :update_counter
+
+  def update_counter
+    post.update(comments_counter: post.comments.size)
+  end
```

Comment:
## Optional
- [ ] Same here I suggest that you use the `after_save` and remember to make `update_counter` private